plugins/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
-if(NOT WIN32)
+if(MSYS OR NOT WIN32)
```

Comment:
`MSYS` is only defined only for 'MSYS Makefiles'... You want to test for `MINGW`, or even better/simpler, just test for `NOT MSVC`

Reply:
Ah thanks, I'll update it. I know very little about all those different Windows build flags, so thanks for pointing me to the right one :)